GOVERNANCE.md

```diff
@@ -11,21 +11,36 @@ The Fluentd community adheres to the following principles:
 
 ## Voting
 
-The Fluentd project employs "organization voting" to ensure no single organization can dominate the project.
+The Fluentd project employs "organization voting" and "Vetoes" to ensure no single organization can dominate the project.
+
+For formal votes, a specific statement of what is being voted on should be added to the relevant GitHub issue or PR. Maintainers should indicate their yes/no vote on that issue or PR, and after a suitable period of time, the votes will be tallied and the outcome noted.
+
+All the following conditions must be satisfied to approve the formal votes.
+
+- No effective objection ballot: See Vetoes, below.
+- At least effective __2-organization__ affirmative vote: See Organization Vote, below.
+- At least effective __3-maintainers__ affirmative vote.
+- At least __2-week__ for voting.
+
```

Comment:
Maybe better to note that ballot can be changed during the voting period.
That rule is expected with further discussion, opinion may be changed from objection to affirmative onr.

Reply:
Thanks for the review!

Would it be good to add it to the next section?

https://github.com/fluent/fluentd/blob/050fe7a83b32565b4a49b59db5502aa8bbe78847/GOVERNANCE.md?plain=1#L33-L37

Reply:
Yes, it is reasonable to do so.

Reply:
Fixed! Thanks!

https://github.com/fluent/fluentd/compare/050fe7a83b32565b4a49b59db5502aa8bbe78847..5c00a9df86f0ac73f6a22a98505efd849cf9ddbf